pgjdbc/src/main/java/org/postgresql/core/v3/ConnectionFactoryImpl.java

```diff
@@ -657,25 +658,27 @@ private void doAuthentication(PGStream pgStream, String host, String user, Prope
             switch (areq) {
               case AUTH_REQ_MD5: {
                 byte[] md5Salt = pgStream.receive(4);
-                if (LOGGER.isLoggable(Level.FINEST)) {
+                if (false && LOGGER.isLoggable(Level.FINEST)) {
```

Comment:
Why the `false &&`?

Reply:
I can probably keep it as is, however, it just bothers me we log credentials in plain text

Reply:
It's likely there to allow debugging the protocol so replacing it with `false &&` wouldn't make sense. Either we'd remove it entirely or put low level security related logging behind some other feature flag.

Either way it does not belong in this PR and should be done separately.

Reply:
I agree it does not belong to the PR, however, the PR does touch the related lines, so I think it would be fine to disable the log for now, and later we can do something with it.